jdaviz/tests/test_subsets.py

```diff
@@ -24,8 +24,8 @@ def test_region_from_subset_2d(cubeviz_helper):
 
     cubeviz_helper.app.get_viewer('flux-viewer').apply_roi(EllipticalROI(1, 3.5, 1.2, 3.3))
 
-    subsets = cubeviz_helper.app.get_subsets_from_viewer('flux-viewer')
-    reg = subsets.get('Subset 1')
+    subsets = cubeviz_helper.app.get_subsets()
+    reg = subsets.get('Subset 1')[0]['region']
```

Comment:
This is good but for posterity there is another way to get this same return:
`cubeviz_helper.app.get_subsets('Subset 1', object_only=True)[0]`
This makes it so that only the region objects are returned for spatial subsets.

Reply:
Definitely good to know! Since these tests are explicitly extracting the region, I think it's useful to see where the regions are coming from in the full dictionary, but this is still great to keep in mind for more general audiences

Reply:
Why not just change it now?

Reply:
> I think it's useful to see where the regions are coming from in the full dictionary

But when we advertise to our users, I think the other syntax is better